js/sum.js

```diff
@@ -0,0 +1,46 @@
+let addBtn = document.getElementById('add-btn');
```

Comment:
What will happen if we move the script tag in the body in the header? 
Is this going to work?

Reply:
I agree. I fixed it using window eventListener('DOMContentLoaded').